### DISCUSSION TITLE
Rendering DNA with style "ribbon"

### DISCUSSION THREAD

**Discussion by JuhaHuiskonen** [BradyAJohnston/MolecularNodes#257] [Q&A]
I am using Molecular Nodes 2.7.4.  When I try to render DNA (from PDB:1bna) the ribbon is not shown, only the bases.  This is using MOL_style_ribbon_nucleic and its output "Ribbon + bases".  How can I visualise DNA with ribbons? 

<img width="363" alt="Screenshot 2023-06-28 at 14 03 15" src="https://github.com/BradyAJohnston/MolecularNodes/assets/20352352/6459975e-e443-4aaf-bbf5-6453f8cda777">

**Comment by BradyAJohnston** [comment]
It seems to not be broken for me and works OK. Which version of Blender are you using? It might be that an older version of Blender the nodes are broken. I have tested on 3.5 and 3.6 and it works OK.

**Comment by JuhaHuiskonen** [comment]
My Blender was still 3.3 so this must have been the reason. Thank you!
